specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/virtualMachine.json

```diff
@@ -500,6 +500,22 @@
             "type": "string",
             "description": "The name of the virtual machine."
           },
+          {
+            "name": "If-Match",
```

Comment:
please add these at the end of the "parameters" to resolve the breaking change flag